core/output/telegram.js

```diff
@@ -0,0 +1,43 @@
+var options = {
+    // Used by the monkey patch to node-telegram-bot-api
+    stopPolling: true
+};
+var TelegramBot = require.safe('node-telegram-bot-api');
+// Adds the missing stopPolling() method
+// https://github.com/yagop/node-telegram-bot-api/pull/51#issuecomment-217395990
+require.safe('monkey-patches-node-telegram-bot-api')(TelegramBot, options);
+
+var bot = null,
+    shim = require('../shim.js'),
+    api = null;
+
+sendMessage = function(message, thread, opts) {
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: ["sendMessage" is not defined.](https://www.codacy.com/app/theprojectshop/Kassy/file/2530115555/issues/source?bid=3282987&fileBranchId=3282994#l14)